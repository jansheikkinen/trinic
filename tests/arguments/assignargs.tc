// Assignment Arguments

enum TestEnum
  ONE = 1,
  TWO,
  THREE = 2,
  FOUR
end
