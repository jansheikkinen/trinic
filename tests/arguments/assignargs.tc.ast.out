
### AST GENERATION ###
Declaration ENUM
  Arg ASSIGN
    Assign_Arg ONE
      Expr LITERAL
        INT 1
    Assign_Arg TWO
    Assign_Arg THREE
      Expr LITERAL
        INT 2
    Assign_Arg FOUR
